analytics-webapps/src/main/webapp/skin/less/analytics.less

```diff
@@ -1,5 +1,20 @@
 @import "./variables.less";
 
+#activeUsers{
+  padding-top: 0px;
+  #activeUsersChildren{
+    display: flex;
+    #activeUsersAnalytics{
+      width: 70%;
```

Comment:
when using flex box, all you need are in [Vuetify CSS Helpers](https://vuetifyjs.com/en/styles/flex/#flex-wrap) and [Vuetify Grid system](https://vuetifyjs.com/en/components/grids/#usage), no need to add useless CSS